docs/feeders/5-openpnp-setup/feeder-openpnp-setup.md

```diff
@@ -6,16 +6,20 @@ If you'd like to start with a fresh config, you can download the `openpnp-config
 
 To get started the first time with your new feeders, you'll need to attach them all to your LumenPnP, and let OpenPnP scan them to save their information.
 
+## Preliminary Steps
+
+If you have a v2 LumenPnP, the default Z height of your nozzle could cause a collision with feeders. We recommend printing and installing updated Z-gantries (shown below) and nozzle rack, which moves your nozzles up 13mm preventing a collision. You can download them [here](https://github.com/opulo-inc/lumenpnp/releases/download/v3.0.4/LumenPnP-STLs-v3.0.4.zip).
```

Comment:
I chose a specific version because it's known-good. The latest release could mean v4, which doesn't work for v2 anymore. This guarantees folks always get the right part!